db/schema.rb

```diff
@@ -182,10 +178,10 @@
     t.string   "login",      limit: 40
     t.string   "name",       limit: 100, default: ""
     t.string   "email",      limit: 100
+    t.string   "city",       limit: 100,                       null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "password",   limit: 40,                        null: false
-    t.string   "city",       limit: 255
+    t.string   "password",   limit: 40
```

Comment:
These changes are not relevant to the current feature. Please, don't include it in the diff.